specs/multicloud/Huawei_Cloud_File_Block.md

```diff
@@ -0,0 +1,931 @@
+    "type": "cloudFS",
+    "region": "cn-north-1",
+    "availabilityZone": "cn-north-1b",
+    "status": "updating",
+    "protocols": [
+        "NFS"
+    ],
+    "encrypted": true,
+    "encryptionSettings": {
+        "DomainId": "<DomainId>",
+        "KmsKeyId": "<KmsKeyId>",
+        "KmsKeyName": "<KmsKeyName>"
+    },
+    "metadata": {
+        "fields": {
+            "CreatedAt": {
+                "Kind": {
+                    "StringValue": "2020-10-15T02:53:47.914879Z"
+                }
+            },
+            "ExportLocation": {
+                "Kind": {
+                    "StringValue": "<ExportLocation>"
+                }
+            },
+            "ExportLocations": {
+                "Kind": {
+                    "StringValue": "<ExportLocation>"
+                }
+            },
+            "Host": {
+                "Kind": {
+                    "StringValue": ""
+                }
+            },
+            "HwSFSType": {
+                "Kind": {
+                    "StringValue": "SFS"
+                }
+            },
+            "IsPublic": {
+                "Kind": {
+                    "BoolValue": false
+                }
+            },
+            "Links": {
+                "Kind": {
+                    "StructValue": {
+                        "fields": {
+                            "href": {
+                                "Kind": {
+                                    "StringValue": "<Links>"
+                                }
+                            },
+                            "rel": {
+                                "Kind": {
+                                    "StringValue": "self"
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "ProjectID": {
+                "Kind": {
+                    "StringValue": "<ProjectID>"
+                }
+            },
+            "ShareID": {
+                "Kind": {
+                    "StringValue": "cf149d37-09f6-45eb-8c73-8713f0abffeb"
+                }
+            },
+            "ShareNetworkID": {
+                "Kind": {
+                    "StringValue": ""
+                }
+            },
+            "ShareType": {
+                "Kind": {
+                    "StringValue": "default"
+                }
+            },
+            "SnapshotID": {
+                "Kind": {
+                    "StringValue": ""
+                }
+            },
+            "VolumeType": {
+                "Kind": {
+                    "StringValue": "default"
+                }
+            }
+        }
+    }
+}
+```
+**Delete File share**
+
+DELETE http://192.168.20.162:8089/v1/94b280022d0c4401bcf3b0ea85870519/file/shares/5f87b9bc90f7e10001fa260c
+
+Response:
+
+```
+{}
+```
+
+## Dependencies
+
+The support for Huawei cloud SFS and EVS will use the GOLANG SDK provided. Here is the reference:
```

Comment:
what is the translation here, specifically for error scenarios ? we have this issue in the object API for AWS etc. too, specifically

1. error response are of 2 types, errors/situations in SODA
2. errors in the actual back-end provider
3. all errors in the actual back-end provider must be passed on to the user via SODA, to enable debugging and fixing

Reply:
I will update this part, basically we want to have SODA generic Error message. The error from backend provider will be logged but what exactly is propagtaed to the end user is the SODA error message which becomes common across providers.